test/org/traccar/ProtocolTest.java

```diff
@@ -98,6 +98,10 @@ protected void verifyAttributes(BaseProtocolDecoder decoder, Object object) thro
         verifyDecodedPosition(decoder.decode(null, null, object), false, true, null);
     }
 
+    protected void verifyAttributesList(BaseProtocolDecoder decoder, Object object) throws Exception {
```

Comment:
Why do you need a new method?

Reply:
This is to verify attributes in the case when decoder.decode returns List<Positions>.
I don't persist it to be a separate method. 
I'm ready to reimplement it if you could advise where it's better to place is.

Reply:
Let's try without changing this base class for now. I believe you can use `verifyPositions` method.